flink-runtime/src/main/java/org/apache/flink/runtime/state/OperatorStateHandle.java

```diff
@@ -38,8 +38,8 @@
 	private final StreamStateHandle delegateStateHandle;
 
 	public OperatorStateHandle(
-			StreamStateHandle delegateStateHandle,
-			Map<String, long[]> stateNameToPartitionOffsets) {
+			Map<String, long[]> stateNameToPartitionOffsets,
```

Comment:
Changing the order here had the purpose of having the same order in `OperatorStateHandle` as in `KeyGroupStateHandle`, which was exactly the other way around :)